docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,789 @@
+        DPU_0_T         37.438      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_1_T         37.563      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_2_T           38.5      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_3_T         38.813      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+     FAN_Sensor         23.201      100.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+ MB_PORT_Sensor         21.813       97.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+MB_TMP421_Local          26.25      135.0      -5.0           140.0          -10.0      False  20230728 06:39:18
+       SSD_Temp           40.0       80.0      -5.0            83.0          -10.0      False  20230728 06:39:18
+   X86_CORE_0_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_PKG_TEMP           41.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+```
+#### Pass/Fail Criteria
+ * Verify warnings are all false
+ * Verify changing the threshold values (high, low, critical high and critical low) and check alarm warning changing into True
+ * Verify changing back the threshold values to original one and check alarm warning changing into False
+
+
+### 1.4 Check DPU Console
+
+#### Steps
+ * Use serial port utility to access console for given DPU.
+ * Get the mapping of serial port to DPU number from platform.json file. 
+ * Get the number of DPU modules from PMON APIs - get_num_modules(). Test is to check for console access for all DPUs.
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: (shows connection to dpu-4 console and offset is 4 for this case)
+
+root@sonic:/home/cisco# cat /proc/tty/driver/serial 
+serinfo:1.0 driver revision:
+0: uart:16550A port:000003F8 irq:16 tx:286846 rx:6096 oe:1 RTS|DTR|DSR|CD|RI
+1: uart:16550A port:00006000 irq:19 tx:0 rx:0 CTS|DSR|CD
+2: uart:16550A port:000003E8 irq:18 tx:0 rx:0 DSR|CD|RI
+3: uart:16550A port:00007000 irq:16 tx:0 rx:0 CTS|DSR|CD
+**4: uart:16550 mmio:0x94040040 irq:94 tx:0 rx:0
+5: uart:16550 mmio:0x94040060 irq:94 tx:20 rx:68
+6: uart:16550 mmio:0x94040080 irq:94 tx:0 rx:0
+7: uart:16550 mmio:0x940400A0 irq:94 tx:0 rx:0
+8: uart:16550 mmio:0x940400C0 irq:94 tx:0 rx:0
+9: uart:16550 mmio:0x940400E0 irq:94 tx:0 rx:0
+10: uart:16550 mmio:0x94040100 irq:94 tx:0 rx:0
+11: uart:16550 mmio:0x94040120 irq:94 tx:0 rx:0**
+12: uart:16550 mmio:0x94040140 irq:94 tx:0 rx:0 CTS|DSR
+13: uart:16550 mmio:0x94040160 irq:94 tx:0 rx:0 CTS|DSR
+14: uart:16550 mmio:0x94040180 irq:94 tx:0 rx:0 CTS|DSR
+15: uart:16550 mmio:0x940401A0 irq:94 tx:0 rx:0 CTS|DSR
+
+root@sonic:/home/cisco# /usr/bin/picocom -b 115200 /dev/ttyS8
```

Comment:
How is ttyS8 mapped for DPU-4 here?

Reply:
Here, ttyS4 to ttyS11 are mapped to DPU0 to DPU4 and hence ttyS8 is DPU-4.
The mapping information would be taken from platform.json